app/src/main/java/io/intrepid/skeleton/utils/RxUtils.java

```diff
@@ -0,0 +1,25 @@
+package io.intrepid.skeleton.utils;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import rx.functions.Action1;
+import rx.functions.Func1;
+import timber.log.Timber;
+
+public class RxUtils {
+
```

Comment:
Maybe add some more functions?
(https://github.com/IntrepidPursuits/MITMobius-android/blob/develop/app/src/main/java/edu/mit/mobius/utils/RxHelper.java)

RxBus is a relay so it never calls onComplete -> (https://github.com/IntrepidPursuits/MITMobius-android/blob/develop/app/src/main/java/edu/mit/mobius/utils/RxRelay.java)